package.json

```diff
@@ -85,6 +86,7 @@
     "ignore-styles": "^5.0.1",
     "jest-cli": "^22.4.3",
     "jest-enzyme": "^6.0.0",
+    "jscodeshift": "^0.11.0",
```

Comment:
I'm curious what your experience with `jscodeshift` was and how you'd compare it to working with just recast?

Reply:
Actually understanding it's API was really confusing for me but it's much more powerful if you can find the right functions/access patterns.

Recasts I think is alot clearer in what it's doing but hard to write.